### ISSUE TITLE
all or peekAll?

### ISSUE THREAD

**Issue by zacstewart** [ember-data/ember-data-filter#6]
The code example for getting away from `store.filter` says to use `store.all`:

``` javascript
// app/controllers/posts.js

export default Ember.Controller.extend({

  posts: Ember.computed(function() {
    return this.store.all('post');
  }),

  filteredPosts: Ember.computed('posts.@each.isPublished', function() {
    return this.get('posts').filterBy('isPublished');
  })
});
```

In doing so, I noticed that `store.all` is also deprecated. Should this example be updated to use `peekAll`?

**Comment by fivetanley** [comment]
Yeah! It'd be great if you have time to make a PR. `peekAll` is the preferred way.

**Comment by pangratz** [comment]
This has been addressed in #7 and #8.
